bundles/org.eclipse.equinox.http.servlet/META-INF/MANIFEST.MF

```diff
@@ -18,7 +18,6 @@ Export-Package: org.eclipse.equinox.http.servlet;version="1.2.0";
  org.eclipse.equinox.http.servlet.dto;version="1.0.0";x-internal:=true,
  org.eclipse.equinox.http.servlet.session;version="1.0.0";x-internal:=true
 Import-Package: javax.servlet;version="[3.1.0,5.0.0)",
- javax.servlet.descriptor;version="[3.1.0,5.0.0)",
```

Comment:
It looks like this is required implicitly I just didn't find the place where a classnotfound occurrs.